pulsar-io/flume/src/test/java/org/apache/pulsar/io/flume/node/TestEnvVarResolverProperties.java

```diff
@@ -18,52 +18,51 @@
  */
 package org.apache.pulsar.io.flume.node;
 
-import java.io.File;
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Rule;
-import org.junit.Test;
 import org.junit.contrib.java.lang.system.EnvironmentVariables;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.File;
 
 public final class TestEnvVarResolverProperties {
     private static final File TESTFILE = new File(
             TestEnvVarResolverProperties.class.getClassLoader()
                     .getResource("flume-conf-with-envvars.properties").getFile());
 
-    @Rule
     public final EnvironmentVariables environmentVariables = new EnvironmentVariables();
```

Comment:
The `org.junit.contrib.java.lang.system.EnvironmentVariables` class depends on the usage of Junit. The usage of EnvironmentVariables class should be adapted if junit is removed from this class and replaced with TestNG. Please address this aspect.

Reply:
Done

Reply:
I don't think that @Rule annotations are supported in TestNG tests. You will have to either rollback the changes to this test class and leave it as it is or replace the functionality provided by EnvironmentVariables class by something else that works with TestNg. 
It might be easier to simply leave this test class as it is even though it uses Junit.

Reply:
Ok, i roll the test class back, PTAL, thanks.